apiary.apib

```diff
@@ -3564,7 +3564,7 @@ Get details for a single work order.
                         + id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string)
                     + started_at: `2019-02-27T10:01:49+00:00` (string)
                     + duration: 3600 (number) - In seconds
```

Comment:
Will we change this to match TimeTracking format? 
`started_at` & `duration` OR  `started_at` & `ended_at`

Reply:
No. This was in the initial proposal but we choose to not implement this immediately. The mobile team only uses the duration.